src/patternfly/components/Table/examples/Table.md

```diff
@@ -1705,12 +1655,293 @@ Note: To apply padding to `.pf-c-table__expandable-row`, wrap the content in `.p
   {{/table-tbody}}
 {{/table}}
 ```
+
+## Controlling text
+
+To better control table cell behavior, PatternFly provides a series of modifiers to help contextually control layout. By default, `thead` cells are set to truncate, whereas `tbody` cells are set to wrap. Both `th` and `td` cells use a set of shared css properties mapped to customizable css variable values. Because only the shared css variables are changed by the modifier selector and not the properties, the modifier can be applied to any parent element up until `.pf-c-table` itself [`thead`, `tbody`, `tr`, `th`, `td`, `.pf-c-table__text`].
+
+<ul>
+  <li>
+    <b>pf-m-wrap:</b> Sets text to wrap. This is the default behavior for <code>tbody</code> cells.
```

Comment:
I think it would be helpful here to add a table of what these modifiers apply to. because im looking here and then im like ok now i have to search for which modifier to use for table header or for table cell, so maybe you could just add the documentation for all these modifiers here